### ISSUE TITLE
Background page of Popups is way too dark (WET 4.0.17)

### ISSUE THREAD

**Issue by RitchieD** [wet-boew/wet-boew#7092]
As of WET 4.0.17 Background page of Popups is way too dark.  I’m assuming this was intentional, can we have way to control that.  I personally don’t like it and I think it even violates Accessibility contrast rules.

http://wet-boew.github.io/v4.0-ci/demos/overlay/overlay-en.html

**Comment by pjackson28** [comment]
Note that it doesn't violate accessibility contrast rules, it was darkened so accessibility contrast rules could consistently be met. Accessibility contrast rules don't apply to any of the content behind the overlay (not intended to be visible, just like when content falls behind a popup).

Here is the actual exception in Success Criterion 1.4.3: "Incidental:  Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement."

As for providing control over the actual colour and darkness, that is another matter (someone else can respond to that).

**Comment by pjackson28** [comment]
Also note that this issue is specific to the Lightbox overlays as the other overlay types don't have the darkened background.

**Comment by andre-gh** [comment]
You should be able to override it with css with `.mfp-bg` (`background-color` and `opacity`), do you want to control the overlay on a case-by-case basis programmatically or is having it part of your CSS sufficient?   It seems to me like changing it would be a theme matter rather than a content defined matter, unless you have a use case where you want to control it variably?

**Comment by RitchieD** [comment]
@andre-gh  css works for me thank you.
